docs/clients/http/protocol.rst

```diff
@@ -4,28 +4,30 @@
 Protocol
 ========
 
-EdgeDB supports GET and POST methods for handling EdgeQL over HTTP
-protocol. Both GET and POST methods use the following fields:
+EdgeDB supports ``GET`` and ``POST`` request methods for handling EdgeQL
+over an HTTP protocol. Both of these methods use the following fields:
 
-- ``query`` - contains the EdgeQL query string
+- ``query`` - contains a string value for an EdgeQL query.
```

Comment:
The new wording is confusing (string value here is the actual query). What's wrong with the original "contains the EdgeQL query string" wording?